app/migrations/repositories/LockRefreshChecker.scala

```diff
@@ -22,8 +22,8 @@ import migrations.LockManager.LockRefreshMarginMillis
 import migrations.TimeUtils
 
 class LockRefreshChecker(private val timeUtils: TimeUtils) {
-  def needsRefreshLock(lockExpiresAt: Date): Boolean =
-    lockExpiresAt == null ||
-      timeUtils.currentTime.compareTo(new Date(lockExpiresAt.getTime - timeUtils.minutesToMillis(LockRefreshMarginMillis))) >= 0
+  def needsRefreshLock(lockExpiresAt: Option[Date]): Boolean =
+    lockExpiresAt.isEmpty ||
+      timeUtils.currentTime.compareTo(new Date(lockExpiresAt.get.getTime - timeUtils.minutesToMillis(LockRefreshMarginMillis))) >= 0
```

Comment:
Nit: We tend not to call `get` on `Option` in production code.
I understand that in this particular case it should not be a problem since for empty `Option` the first part of the expression is `true` and for optimization the second part is not executed at all. But I am wondering if it could be done in a single expression (like using `map` and then `getOrElse(false)` ?

Reply:
:+1:  np